artifacts.py

```diff
@@ -22,141 +24,179 @@
 logger = logging.getLogger(__name__)
 
 
-class v1_do_artifacts_connector:
-    def __init__(self):
-        self.template_path = os.path.abspath("./templates")
-        self.env = Environment(loader=FileSystemLoader(self.template_path))
+def command_handler(error_context: str):
```

Comment:
Added a decorator to standardize endpoint error handling

Reply:
It's really just a way to standardize error handling so we aren't repeating our try/catch logic for every endpoint. (I realized I was just copy-and-pasting code!) We pass in the error context as an endpoint-specific helper (so we know which endpoint failed).